app/assets/v2/js/pages/bulk_payout.js

```diff
@@ -166,7 +166,12 @@ $(document).ready(function($) {
       _alert('You do not have any transactions to payout.  Please add payees to the form.', 'error');
       return;
     }
-
+    $('.username').each(function() {
+      if ($(this).text().trim() === '' || $(this).text() === null || $(this).text() === '@') {
```

Comment:
It should be `$(this).text().trim() === '@'`

Reply:
@rahulpat ^